### ISSUE TITLE
Fake XP drops

### ISSUE THREAD

**Issue by Noxally** [Clownvin/SoulCraft#1]
Upon uptading from Soulcraft 1.0.0 to 2.0.0, every block broken drops a "phantom" xp orb that is multicolored and cannot be collected, which leads to issues in autobreakers etc as non despawning XP orbs that are shifting between every possible rgb color cause a bit of lag after a time. Downgrading to 1.0.0 fixed the issue

**Comment by Clownvin** [comment]
I'll check it out.

**Comment by DWX12** [comment]
Yeah, I'm experiencing the same issue. Believe it to be a problem with this mods experience gathering system when set to that option in the config. Setting the xp gathering option to anything else stops the orbs from appearing.
